packages/cli-lib/http.js

```diff
@@ -147,9 +150,22 @@ const createGetRequestStream = ({ contentType }) => async (
         },
         json: false,
       });
-      req.on('error', reject);
+      req.on('error', error => {
```

Comment:
Curious on this change.  aren't the two ways equivalent?

Reply:
I think this is a left over from some debugging. I'll go back to the terser and more direct version.